modules/ROOT/pages/deploy-message-flows.adoc

```diff
@@ -10,28 +10,32 @@ To deploy a message flow, click the *Deploy* button.
 
 The message flow is deployed to Mule runtime engine, enabling you to:
 
-* Start receiving messages from your partner
-* Use the *Activity* page to track the processed messages
+* Start receiving messages from your partner.
+* Use the *Activity* page to track the processed messages.
 
 == Test a Deployed Message Flow
 
 After you deploy the message flow, send test messages to verify that:
 
 * You can receive messages and acknowledgments from your partners.
-* The DataWeave map works.
-* Your backend applications receive the transformed messages.
-
-Test your message flow in your Partner Manager sandbox environment by sending a test file to the receiving endpoint:
-
-. In the sidebar, select *Activity* and the transmission to test.
-. Scan for an exclamation point (!) in any of the boxes next to the displayed flow stages and address any issues.
-. If the system is set up to view payloads, review the received payload by clicking *View payload* next to the *Received transmission* section.
+* The DataWeave map transformed the payload correctly.
+* Your backend applications received the transformed messages.
+
+Test your message flow in the Partner Manager sandbox environment by sending a test file to the receiving endpoint:
+
+. In the sidebar, select *Activity*.
+. Search for the transmission by specifying the desired filters.
+. In the transmission search results, you can do the following:
+* Verify that the transmission was sent or received. 
+* Verify that Partner Manager sent the transmission to the correct endpoint.
+* If TA1 and functional acknowledgments (999 or 997) were set, review and confirm that they are present in the transmission.
+. Click the transmission to view more summary information. 
++
+Here, you can verify whether Partner Manager routed the transmission to the correct message flow and whether it sent a message disposition notification (MDN).
```

Comment:
MDN is applicable only for AS2. Perhaps we can mention that at the end of the statement - something like below: 

you can verify whether Partner Manager routed the transmission to the correct message flow and whether it sent a message disposition notification (MDN) if the transmission involves AS2 endpoints.